profiler/src/main/java/eu/clarin/switchboard/profiler/api/LanguageCode.java

```diff
@@ -0,0 +1,207 @@
+            .put("la", "lat")
+            .put("lb", "ltz")
+            .put("lg", "lug")
+            .put("li", "lim")
+            .put("ln", "lin")
+            .put("lo", "lao")
+            .put("lt", "lit")
+            .put("lu", "lub")
+            .put("lv", "lav")
+            .put("gv", "glv")
+            .put("mk", "mkd")
+            .put("mg", "mlg")
+            .put("ms", "msa")
+            .put("ml", "mal")
+            .put("mt", "mlt")
+            .put("mi", "mri")
+            .put("mr", "mar")
+            .put("mh", "mah")
+            .put("mn", "mon")
+            .put("na", "nau")
+            .put("nv", "nav")
+            .put("nd", "nde")
+            .put("ne", "nep")
+            .put("ng", "ndo")
+            .put("nb", "nob")
+            .put("nn", "nno")
+            .put("no", "nor")
+            .put("ii", "iii")
+            .put("nr", "nbl")
+            .put("oc", "oci")
+            .put("oj", "oji")
+            .put("cu", "chu")
+            .put("om", "orm")
+            .put("or", "ori")
+            .put("os", "oss")
+            .put("pa", "pan")
+            .put("pi", "pli")
+            .put("fa", "fas")
+            .put("pl", "pol")
+            .put("ps", "pus")
+            .put("pt", "por")
+            .put("qu", "que")
+            .put("rm", "roh")
+            .put("rn", "run")
+            .put("ro", "ron")
+            .put("ru", "rus")
+            .put("sa", "san")
+            .put("sc", "srd")
+            .put("sd", "snd")
+            .put("se", "sme")
+            .put("sm", "smo")
+            .put("sg", "sag")
+            .put("sr", "srp")
+            .put("gd", "gla")
+            .put("sn", "sna")
+            .put("si", "sin")
+            .put("sk", "slk")
+            .put("sl", "slv")
+            .put("so", "som")
+            .put("st", "sot")
+            .put("es", "spa")
+            .put("su", "sun")
+            .put("sw", "swa")
+            .put("ss", "ssw")
+            .put("sv", "swe")
+            .put("ta", "tam")
+            .put("te", "tel")
+            .put("tg", "tgk")
+            .put("th", "tha")
+            .put("ti", "tir")
+            .put("bo", "bod")
+            .put("tk", "tuk")
+            .put("tl", "tgl")
+            .put("tn", "tsn")
+            .put("to", "ton")
+            .put("tr", "tur")
+            .put("ts", "tso")
+            .put("tt", "tat")
+            .put("tw", "twi")
+            .put("ty", "tah")
+            .put("ug", "uig")
+            .put("uk", "ukr")
+            .put("ur", "urd")
+            .put("uz", "uzb")
+            .put("ve", "ven")
+            .put("vi", "vie")
+            .put("vo", "vol")
+            .put("wa", "wln")
+            .put("cy", "cym")
+            .put("wo", "wol")
+            .put("fy", "fry")
+            .put("xh", "xho")
+            .put("yi", "yid")
+            .put("yo", "yor")
+            .put("za", "zha")
+            .put("zu", "zul")
+            .build();
```

Comment:
Just as a note, we do something similar in the VLO so perhaps there's a potential for sharing a static resource for this (but based on external definitions). But as it won't change, there isn't much risk or cost to having this implemented in two places.

Reply:
I found a simpler way, relying on standard library.